fakefs.gemspec

```diff
@@ -19,4 +19,12 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '>= 10.3'
   spec.add_development_dependency 'rspec', '~> 3.1'
   spec.add_development_dependency 'rubocop', '~> 0.82.0'
+
+  # It is good practice to have all deps listed, they will appear in the Gemfile.lock and allow reproducible builds
+  # For an unknown reason the github CI does not like when "pathname" is added
```

Comment:
so does this still solve your issue when pathname is still missing ?

Reply:
Actually my use case just needs `irb` to be included 👼🏻 
I just decided to be consistent with the so called best practice ... but it does not work well with pathname :/ 

When I also add `pathname` the error is:
> /<gem_home>/bundler/runtime.rb:304:in `check_for_activated_spec!': You have already activated pathname 0.2.1, but your Gemfile requires pathname 0.3.0. Since pathname is a default gem, you can either remove your dependency on it or try updating to a newer version of bundler that supports pathname as a default gem. (Gem::LoadError)


Fun thing the current failure on `ruby-head` is somewhat related to this best practice as bundler is failing to load `mutex_m` from stdlib on ruby 3.4.0 :) (but this is not a `fakefs` issue)